### PR TITLE
add filter press model

### PR DESCRIPTION
**Pull request by kurbansitterley**
## Summary/Motivation:

Add filter press module

## Changes proposed in this PR:
- add filter press model and tests
- update yaml

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by andrewlee94 — DISMISSED** [review]
Looks OK.

**Comment by andrewlee94** [comment]
@kurbansitterley This PR has some non-trivial conflicts that need to be fixed.

**Review by luohezhiming — DISMISSED** [review]
LGTM

**Review by andrewlee94 — DISMISSED** [review]

File: watertap/data/techno_economic/filter_press.yaml [hunk 1]
```diff
@@ -26,130 +26,6 @@ default:
       value: 433972
       units: USD_2007
   removal_frac_mass_solute:
-    aluminum:
```
Comment (andrewlee94) [inline watertap/data/techno_economic/filter_press.yaml]: Just to check - the removal of all these parameters was intended?

**Review by andrewlee94 — APPROVED** [review]

**Review by luohezhiming — APPROVED** [review]

File: watertap/data/techno_economic/filter_press.yaml (hunk 1 above)
Comment (kurbansitterley) [reply watertap/data/techno_economic/filter_press.yaml]: Yes correct. I had initially copied the evaporation pond yaml and forgot to pare it down.

**Pull request merged**